hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/HttpBackendResolver.java

```diff
@@ -51,7 +61,19 @@
           event, entityBuilder, HttpSemanticConventionUtils.getAttributeKeysForHttpMethod());
       setAttributeForFirstExistingKey(
           event, entityBuilder, HttpSemanticConventionUtils.getAttributeKeysForHttpRequestMethod());
-      return Optional.of(entityBuilder.build());
+
+      Map<String, AttributeValue> enrichedAttributes = new HashMap<>();
+      String httpOperation =
```

Comment:
This has been already computed prior, can we just use
```
String operation = httpRequest.map(Request::getMethod).orElse(null);
```

Reply:
Same for GRPC, we are doing the same lookup computation twice. However, in GRPCBackendResolver, we are not using `evet.getGrpc()` request, so it can be done as part of follow-up PR. But, here in HTTPBackendResolver, we are already using `event.getHttp()` request, so we don't have to re-compute.